src/v/test_utils/runfiles.cc

```diff
@@ -17,8 +17,13 @@ get_runfile_path([[maybe_unused]] std::string_view path) {
 #ifdef BAZEL_TEST
```

Comment:
When/where is `BAZEL_TEST` set?

Reply:
The bazel build file sets this

Reply:
https://github.com/redpanda-data/redpanda/blob/3e1763f15b7a17057b25b858e27feca6ccf0dc3d/src/v/test_utils/BUILD#L145-L149

Reply:
So this is so that it builds in cmake too, right? H/e in cmake these runfiles methods will always return null, right? Are they expected to guard calls to these methods with the same `ifdef` stuff?

Reply:
Yes cmake always returns `std::nullopt` which means the callers need to handle falling back to how to load the file in CMake.

Here is an example of how it's used:

https://github.com/redpanda-data/redpanda/blob/3e1763f15b7a17057b25b858e27feca6ccf0dc3d/src/v/serde/avro/tests/parser_test.cc#L116-L119

The `value_or` case is CMake

Reply:
Thanks. I was a bit lost as I didn't review the prior change.